karma.conf.js

```diff
@@ -0,0 +1,46 @@
+module.exports = function (config) {
+  config.set({
+
+    port: 9876,
+
+    logLevel: config.LOG_INFO,
+
+    autoWatch: true,
+    singleRun: false,
+    colors: true,
+
+    plugins: [
+      'karma-jasmine',
+      'karma-sinon',
+      'karma-spec-reporter',
+      'karma-chrome-launcher',
+      'karma-firefox-launcher',
+      'karma-browserify'
+    ],
+
+    browsers: ['Chrome'],
```

Comment:
We should add PhantomJS here as well so we can get quick sanity checks from having our tests run in CLI.